core/benches/kura.rs

```diff
@@ -42,7 +43,8 @@ async fn measure_block_size_for_n_validators(n_validators: u32) {
         iroha_core::kura::Kura::new(iroha_config::kura::Mode::Strict, dir.path(), false).unwrap();
     let _thread_handle = iroha_core::kura::Kura::start(kura.clone());
 
-    let mut wsv = WorldStateView::new(World::new(), kura);
+    let query_handle = LiveQueryStore::test().start();
+    let mut wsv = WorldStateView::new(World::new(), kura, query_handle);
```

Comment:
Why might what to reconceptualize what `WSV` is, because currently it contains slightly related things (blocks, data base, wasm engine, query service and so on).

Don't think it should be done in this PR.

Reply:
Yeah, and now I decided to put `query_handle` into it. I tried keeping it separate but it's just too much pain